index.js

```diff
@@ -0,0 +1,152 @@
+const express = require('express');
+const knex = require('knex');
+const knexConfig = require('./knexfile');
+const db = knex(knexConfig.development);
+const server = express();
+server.use(express.json());
+
+// PROJECTS
+
+//GET
+
+server.get("/projects", (req, res) => {
+    db("projects")
+      .then(projects => {
+          res.status(200).json(projects)
+      })
+      .catch(err => {
+          res.status(500).json(err)
+      });
```

Comment:
There's a lot of repetition with your `.then`s and `.catch`s. You could probably easily reduce this with a helper function. For example:
```javascript
const handleData = res => promise => {
  promise
    .then(data => res.status(200).json(data))
    .catch(err => res.status(500).json(err));
};
```
and usage:
```javascript
server.get('/projects', (req, res) => {
  handleData(res)(db('projects'));
});
```

Reply:
I haven't tested this, but I'm assuming it'll work.